salvo/install_deps.sh

```diff
@@ -3,7 +3,6 @@
 if [ ${UID} -ne 0 ]
 then
   echo "This script needs root priviliges to install dependencies. Continuing may elicit failures from tests"
```

Comment:
This script isn't just used in CI, I believe it is also used as the script to install Salvo dependencies locally when starting to develop Salvo:

https://github.com/envoyproxy/envoy-perf/tree/main/salvo#dependencies

Will the script still work for that purpose after the changes here? Or should we have a separate script for CI?

Reply:
@mum4k You're right. It may not work locally on the machine has not `docker.io` or `docker-ce-cli`. I think a separate CI script is a better option. I will create another script named `install_deps_ci.sh` and keep this scripts unchanged.